telegram/ext/handler.py

```diff
@@ -17,6 +17,13 @@
 # You should have received a copy of the GNU Lesser Public License
 # along with this program.  If not, see [http://www.gnu.org/licenses/].
 """This module contains the base class for handlers as used by the Dispatcher."""
+import inspect
+import sys
+
+from telegram.ext import Context
```

Comment:
Do this instead:
```python
from .context import Context
```
(though it might be irrelevant if you'd move the `Context` definition into this file.